src/index.html

```diff
@@ -4,11 +4,37 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Webpack Exercise</title>
+  <title>Leader Board</title>
 
 </head>
 <body>
-  <h1>Hello webpack!</h1>
+  <main>
+    <h1>Leaderboard</h1>
+    <div class="wrapper">
+      <section class="list">
+        <div class="h-b">
+          <h2>Recent Scores</h2>
+          <button type="button">Refresh</button>
+        </div>
+
+        <ul>
+          <li>Name: 100</li>
+          <li>Name: 200</li>
+          <li>Name: 300</li>
+          <li>Name: 400</li>
+        </ul>
```

Comment:
- [ ] Great job so far :clap:  However You must use a JavaScript array to store these Scores then create a function to render through the array as you did with your To-do list project.
- [ ] Use ES6 modules to write modular JavaScript with proper ES6 syntax  and
- [ ] Don't forget to us `import` and `export` to import it as a module

eg: 

``` 
 const array = [
  { x: 'name', y: 100 },
 { x: 'name', y: 100 }
];


export const render = (arr) => {
  arr.forEach((game) => {
    ......
  });
};

```